python/ray/tune/schedulers/pbt.py

```diff
@@ -361,6 +361,17 @@ def __init__(
         require_attrs: bool = True,
         synch: bool = False,
     ):
+        if not _use_storage_context():
+            raise RuntimeError(
+                "Due to breaking API changes, PBT does not work with the old "
+                "persistence mode (enabled via RAY_AIR_NEW_PERSISTENCE_MODE=0). "
+                "Migrate your script to use the new APIs instead and disabled the "
```

Comment:
I think we should just not mention this env var -- all user code even w/o PBT will already error, so we should just tell them: Please migrate to the new APIs to use PBT. 

And link to this issue: `https://github.com/ray-project/ray/issues/37868`

Reply:
SGTM

Reply:
Hm I guess this error message only comes up when someone actually used that var?